test/gtest/ucp/test_ucp_atomic.cc

```diff
@@ -5,229 +5,173 @@
 * See file LICENSE for terms.
 */
 
-#include <common/test.h>
-
 #include "test_ucp_atomic.h"
 extern "C" {
-#include <ucp/core/ucp_context.h>
-}
-
-std::vector<ucp_test_param>
-test_ucp_atomic::enum_test_params(const ucp_params_t& ctx_params,
-                                  const std::string& name,
-                                  const std::string& test_case_name,
-                                  const std::string& tls)
-{
-    std::vector<ucp_test_param> result;
-    generate_test_params_variant(ctx_params, name,
-                                 test_case_name, tls, UCP_ATOMIC_MODE_CPU, result);
-    generate_test_params_variant(ctx_params, name,
-                                 test_case_name, tls, UCP_ATOMIC_MODE_DEVICE, result);
-    generate_test_params_variant(ctx_params, name,
-                                 test_case_name, tls, UCP_ATOMIC_MODE_GUESS, result);
-    return result;
-}
-
-void test_ucp_atomic::init() {
-    const char *atomic_mode =
-                    (GetParam().variant == UCP_ATOMIC_MODE_CPU)    ? "cpu" :
-                    (GetParam().variant == UCP_ATOMIC_MODE_DEVICE) ? "device" :
-                    (GetParam().variant == UCP_ATOMIC_MODE_GUESS)  ? "guess" :
-                    "";
-    modify_config("ATOMIC_MODE", atomic_mode);
-    test_ucp_memheap::init();
+#include <ucp/core/ucp_types.h> // for atomic mode
```

Comment:
use `/**/` or change the comment in
https://github.com/openucx/ucx/blob/18616602fee2e5f3d26af20b462915a9b53fbc64/test/gtest/ucp/test_ucp_rma.cc#L12